codegen/render/proto_types_renderer.go

```diff
@@ -33,7 +33,7 @@ func RenderProtoTypes(grp Group) ([]OutFile, error) {
 func (r ProtoCodeRenderer) RenderProtoHelpers(grp Group) ([]OutFile, error) {
 
 	// only render proto helpers for proto groups in the current module
-	if !grp.HasProtos() || grp.Module != r.GoModule {
```

Comment:
The second clause of this statement would always evaluate to `false`.

Reply:
Are you sure? even in other projects. I don't want downstream projects broken based on this change

Reply:
In this project yes, since `Module` of the `ProtoCodeRenderer` is set to the one of the `Group` when we construct it (see [here](https://github.com/solo-io/skv2/blob/a3079f6f4f9e0263b004d0ee32cf22330c863574/codegen/render/proto_types_renderer.go#L26).

Hadn't considered that the type is exported, though. Is it directly used in other projects? I don't think it is usable outside of this package: there is no way to construct a valid instance since the embedded `templateRenderer` is private and is needed for it to work AFAICT:

```go
type ProtoCodeRenderer struct {
	templateRenderer

	// the go module of the project
	GoModule string

	// the relative path to the api dir
	// types will render in the package <module>/<apiRoot>/<group>/<version>
	ApiRoot string
}
```

I could:
- just revert this change, or
- export the `templateRenderer` type, or
- make the `ProtoCodeRenderer` private

Don't feel strongly either way, wdyt?

Reply:
Fine as is, thanks for the explination!